src/parser/retail/modules/items/dragonflight/EchoingTyrstone.tsx

```diff
@@ -0,0 +1,219 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import { formatNumber, formatPercentage } from 'common/format';
+import ItemLink from 'interface/ItemLink';
+import { CooldownIcon, HasteIcon } from 'interface/icons';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { ApplyBuffEvent, HealEvent, Item } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringItemValueText';
+import ItemPercentHealingDone from 'parser/ui/ItemPercentHealingDone';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import Statistic from 'parser/ui/Statistic';
+
+const ECHOING_TYRSTONE_COOLDOWN = 120000; // 120sec
+
+// Need a better way to do this. The calculateSecondaryStat functions don't seem to line up
```

Comment:
looks like our stat scaling multipliers are out of date

Reply:
Even updating the values (which are very minor fractional changes) the values don't line up. Is on-use scaling different?
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/1750811/424444b9-9351-4091-81e9-7d3848930efa)
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/1750811/f6ce0942-f06d-4bf3-8f6d-3fbeb0b6bdc4)

Reply:
it may be using a different curve. i don't recall offhand how to find the scaling curve that a spell uses

Reply:
okay so part of the issue is that wowhead scaling is wrong. im checking the calculated value against what we compute with the trinket multiplier table and will see what i find

## Wowhead

![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/4909458/ed928354-7dd8-46c1-8473-175c940ca2d4)
https://www.wowhead.com/item=207552/echoing-tyrstone?ilvl=483

## In-Game
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/4909458/122efd72-763e-40c5-82e6-c914b9f73eaf)

Reply:
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/4909458/eef8eb08-245e-41a0-9f45-93199a168b4c)

this is the 483 version. the 273 (in-game, previous comment) vs 272 shown here is just down to rounding (wow always rounds up in tooltips iirc)

Reply:
for the heal value, we need this table, ugh https://github.com/simulationcraft/simc/blob/dragonflight/engine/dbc/generated/rand_prop_points.inc

the formula is just `base_value * (damage_secondary(actual ilvl) / damage_secondary(base ilvl))` where `damage_secondary` is the 3rd column (its `ilvl, damage_replace_stat, damage_secondary`)

we *almost never* need this formula because we can read the healing/damage events but tyrstone's cap obviously matters a bit

i can copy it in, but the constants (and stat scaling) will change again next expansion (the formula probably won't though)